WordPress/Classes/ViewRelated/Comments/CommentModerationBar.swift

```diff
@@ -130,3 +177,15 @@ private extension UIButton {
     }
 
 }
+
+// MARK: - UIView Extension
+
+private extension UIView {
+    func configureAsDivider() {
+        backgroundColor = .textSubtle
```

Comment:
Checked Zeplin, and I think this should be `.systemGray`.

<img width="677" alt="Screen Shot 2021-09-24 at 18 46 33" src="https://user-images.githubusercontent.com/1299411/134669362-489627f0-941a-430f-a891-71986d34e4c3.png">

It's really close to `textSubtle`, but the system gray looks a little bit more subtle / light. (Also, it was tough to click the divider 😅 .)

```suggestion
        backgroundColor = .systemGray
```

Reply:
I'll most likely have to update the colors anyway once Miklic reviews it, so I'll do it all in one go later.